### ISSUE TITLE
[Feature Request] - Support active mode

### ISSUE THREAD

**Issue by devbydav** [veeso/suppaftp#4]
## Description

suppaftp currently supports passive mode only, which is the most commonly used. Sometimes active mode is needed, for example with servers behind a firewall that doesn't open port for data connections.

I have already started implementing it because I needed it, but it would be great to add the feature to suppaftp! I will be happy to make any changes to fit in.

### Changes

Support active mode

**Comment by veeso** [comment]
It's okay for me!

Regarding the implementation in the client structure, I would make a method like this

```rust
pub fn active_mode(mut self) -> Self {
    self.mode = Mode::Active;
}
```

and by default mode is set to `Mode::Passive`

**Comment by veeso** [comment]
Merged into 0.4.2

Thanks again for the amazing contribution!
